### ISSUE TITLE
Julia should read a config file when launching

### ISSUE THREAD

**Issue by ufechner7** [JuliaLang/julia#45455]
Currently you can configure the Julia parameters only as command line parameters.

Please, add the option to create a julia-config file that julia reads automatically at launch as alternative.

Background:
Currently I need to provide a bash script and a .bat script to launch julia with the correct parameters on different operating systems.
It would be easier if I would have to provide only one config file.

In addition to the current command line options this config file should allow different options per julia version.

**Comment by Heptazhou** [comment]
you mean `~/.julia/config/startup.jl`?
if yes then it is weird that you do not know it

**Comment by gbaraldi** [comment]
I think he means somewhere where you put things like `JULIA_NUM_THREADS` or other things that might be environment variables

**Comment by ufechner7** [comment]
No, this is not what I mean. I don't think I can specifiy a system image in startup.jl.

**Comment by ufechner7** [comment]
In the moment I have this startup file:
```
#!/bin/bash

if [[ $(basename $(pwd)) == "bin" ]]; then
    cd ..
fi

julia_version=$(julia --version | awk '{print($3)}')
julia_major=${julia_version:0:3}

export EDITOR="gedit"

if test -f "bin/kps-image-${julia_major}.so"; then
    echo "Found system image!"
    julia -J  bin/kps-image-${julia_major}.so -t 1 --project
else
    julia --project
fi
```
written in a bash and a separate startup file in .bat language for Windows.

I think Julia should be able do do this natively without any extra scripts.

Perhaps these options could also become part of Project.toml

**Comment by simonbyrne** [comment]
#36843 proposed making sysimg an environment variable, but given the plethora of options being added, being somewhat more systematic about this would be helpful.
